docs/source/howto/workflows/pw/relax.md

```diff
@@ -2,4 +2,101 @@
 
 # `PwRelaxWorkChain`
 
-*To be added.*
+As with all work chains, the best way to get started with running the `PwRelaxWorkChain` is *via* the `get_builder_from_protocol()` method.
+
+First, load the AiiDA orm and activate your profile:
+
+```python
+from aiida import orm, load_profile
+
+load_profile()
+```
+
+Next, load the `PwRelaxWorkChain` and the structure/code you want to run:
+
+```python
+from aiida_quantumespresso.workflows.pw.relax import PwRelaxWorkChain
+
+code = orm.load_code('qe-7.2-pw@localhost')
+structure = orm.load_node(1)  # FCC Silicon
```

Comment:
Why not make it actually runnable?
```suggestion
```python
from ase.build import bulk
from aiida_quantumespresso.workflows.pw.relax import PwRelaxWorkChain

code = orm.load_code('qe-7.2-pw@localhost')
structure = orm.StructureData(ase=bulk('Si', 'diamond', 5.43))  # FCC Silicon
```

Reply:
Although having the structure is a bonus, in the end they probably still need to adapt the code ID. ^^